### PR TITLE
add KansyFamily.net

### PR DESCRIPTION
**Pull request by KansyYT**
Laby.net server liste alles geaddet

## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by KansyYT** [comment]
All update

File: minecraft_servers/kansyfamily/background.png
Comment (DoJapHD) [inline minecraft_servers/kansyfamily/background.png:1]: Compress image

File: minecraft_servers/kansyfamily/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/kansyfamily/background@2x.png:1]: Compress image

File: minecraft_servers/kansyfamily/manifest.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  
+  "social": {
+    "web": "https://kansyfamily.net",
+    "twitter": "kansyfamilyNet",
+    "discord": "https://discord.gg/kansyfamily",
+    "tiktok": "kansyfamily"
+  },
+
+  "gamemodes": {
+    "kansyAttack": {
```
Comment (DoJapHD) [inline minecraft_servers/kansyfamily/manifest.json:19]: All lowerkey

File: minecraft_servers/kansyfamily/manifest.json
Comment (DoJapHD) [inline minecraft_servers/kansyfamily/manifest.json:1]: Remove all empty lines

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by KansyYT** [comment]
Update the image and Remove all empty

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**